plugins/database_api/include/golos/plugins/database_api/state.hpp

```diff
@@ -4,14 +4,14 @@
 #include <golos/chain/account_object.hpp>
 #include <golos/chain/steem_objects.hpp>
 #include <golos/chain/steem_object_types.hpp>
-#include <golos/plugins/database_api/api_objects/witness_api_object.hpp>
 #include "forward.hpp"
 #include <golos/api/extended_account.hpp>
 
 namespace golos {
     namespace plugins {
         namespace database_api {
 
+            using golos::protocol::asset;
```

Comment:
For what?

Reply:
this using was in witness_api_object file that i have moved to other plugin so i needed to restore some of using's